common/views/themes/default.js

```diff
@@ -120,6 +131,20 @@ type SpaceProperty =
   | 'left'
   | 'right';
 
+function hexToRgb(hex: string) {
```

Comment:
Added this in order to be able to use `rgba` with variables without the need for the 'polished' library (as is possible with scss), but I no longer think this is necessary with the base/light/dark system, so I'll get rid of it

Reply:
we still seem to be using `lighten` from polished, but hopefully we'll be able to get rid of that (and save 10k) with the logic in this pr (but I thought it made sense to do that in a different pr).

Reply:
Actually, looks like `hexToRgb` might be useful. But we can add it when we get round to making these buttons.

![image](https://user-images.githubusercontent.com/1394592/86943669-a3e44280-c13e-11ea-89db-60065ddf36a9.png)